atlascope/core/models/waypoint.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib import admin
+from django.contrib.gis.db import models as gis_models
+from django.db import models
+
+
+class Waypoint(models.Model):
+    location = gis_models.PointField(null=True)
+    zoom = models.IntegerField(null=True)
+
```

Comment:
Wouldn't we want a text/char field on each waypoint, too? At each zoomed location, there's some bit of important information that should be shared in the tour and displayed when that waypoint is active.

Reply:
I don't think we know yet what (if anything) we'll want to be displayed during an executed tour, so the best thing right now is to keep the code as simple as possible. We will revisit this when we see what it's like to "run" a tour, and at that point we may add more fields here.